### ISSUE TITLE
CRNN in Tensorflow

### ISSUE THREAD

**Issue by pribadihcr** [AKSHAYUBHAT/DeepVideoAnalytics#105]
Hi @AKSHAYUBHAT,

Is this repo https://github.com/TJCVRS/CRNN_Tensorflow, could replace the current Pytorch's CRNN ?

**Comment by AKSHAYUBHAT** [comment]
Possibly, but PyTorch is good enough and existing CRNN implementation is well debugged. I moved from Caffe version of CTPN to TF version since Caffe is not well supported. But in this process the accuracy of the detector took a hit. As a result I would recommend sticking with the PyTorch version for CRNN.
